server/controllers/UserController.js

```diff
@@ -99,37 +98,118 @@ class UserController {
       });
 
       if (!user) {
-        res.status(404).send({
+        return res.status(404).json({
           status: 'failure',
-          message: 'User not found'
+          data: {
+            statusCode: '404',
+            message: 'User not found'
+          }
         });
-        return;
       }
 
-      await User.update({
-        password: PasswordManager.hashPassword(newPassword)
-      }, {
-        where: {
-          email
+      await User.update(
+        {
+          password: PasswordManager.hashPassword(newPassword)
+        },
+        {
+          where: {
+            email
+          }
         }
-      });
+      );
 
-      res.status(200).send({
+      res.status(200).json({
         status: 'success',
-        message: 'Password has been successfully updated. Kindly login.'
+        data: {
+          statusCode: '200',
+          message: 'Password has been successfully updated. Kindly login.'
+        }
       });
     } catch (error) {
       if (error.name === 'TokenExpiredError' || error.name === 'JsonWebTokenError') {
         return res.status(401).send({
           status: 'failure',
-          message: error.name
+          data: {
+            statusCode: '200',
```

Comment:
please check your status code. there is error 401 and 200.

Reply:
Thanks for your observation. Lemme fix it